docs/config/README.md

```diff
@@ -342,6 +347,9 @@ The module will be shown if any of the following conditions are met:
 | ---------- | ------- | -------------------------------------------------------- |
 | `symbol`   | `"🐍 "` | The symbol used before displaying the version of Python. |
 | `disabled` | `false` | Disables the `python` module.                            |
+| `use_pyenv` | `false` | Use pyenv to get Python version                            |
```

Comment:
I think a more appropriate name might be `pyenv_version_name` since the default implementation does identify the python version used by `pyenv`. The difference here would be that it would show the version name extracted by pyenv.

Thoughts?

Reply:
The default implementation doesn't use `pyenv` to get the Python version, but I agree `pyenv_version_name` is more clear

Reply:
Right, the default implementation doesn't use `pyenv`. What I mean to say is that assuming you have installed everything correctly, the version of Python being shimmed by `pyenv` will output the version that is defined by `pyenv` globally or locally. 

We should be ready to merge as soon as the config value's name is updated. 🙂